app/src/main/res/values/strings.xml

```diff
@@ -130,7 +130,7 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="sleep_desc">Meditations while having trouble in sleeping</string>
     <string name="depressed_desc">Meditations when you feel depressed</string>
     <string name="happiness_desc">Meditations while you are in a happy mood</string>
-    <string name="focus_game_one_info">The Rocket Moves Up and Down according to your brain activity. Relax to travel forward or concentrate to travel back.</string>
+    <string name="focus_game_one_info">The rocket moves up and down according to your brain activity. Concentrate to fly up or relax to fly down.</string>
```

Comment:
I saw this in #395 Why don't you properly align branches before making PRs? I've told this many time.

Reply:
Ah! Actually both the PRs happened to be from the sane branch. I was checking for these things only for which didn't request for a review :sweat_smile: . Will update.

Reply:
@CloudyPadmal This change from #395 didn't affect the codebase https://github.com/fossasia/neurolab-android/blob/development/app/src/main/res/values/strings.xml

Reply:
As a result of which this appears to be a change in this PR.

Reply:
That's strange. Why didn't that change make it through? :thinking: